pkg/storage/image.go

```diff
@@ -616,6 +616,8 @@ func splitDockerDomain(name string) (domain, remainder string) {
 	return
 }
 
+// ResolveNames resolves an image name into a storage image ID or a fully-qualified image name (domain/repo/image:tag).
+// Will only return an empty slice if err == nil.
```

Comment:
Sorry, I had a typo in my earlier suggestion :/, should be:

> Will only return an empty slice if err != nil.

Reply:
I should have caught that, my bad. Fixed now